Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -122,24 +151,60 @@ props.buildReportOrder.each{ buildReportFile ->
 		}
 	}
 
+    buildRecords += USSBuildRecords
+
 	def count = 0
+    def USSCount = 0	
 
 	// adding files and executes with outputs to Hashmap to remove redundant data
-	buildRecords.each{ buildRecord ->
-		if (buildRecord.getOutputs().size() != 0) {
-			buildRecord.getOutputs().each{ output ->
-				count++
-				def (dataset, member) = getDatasetName(output.dataset)
-				buildOutputsMap.put(new DeployableArtifact(member, output.deployType), [dataset, buildRecord])
-			}
+	buildRecords.each{ buildRecord ->	
+        if (buildRecord.getType()=="USS_RECORD") {
+            if (!buildRecord.getAttribute("outputs").isEmpty()) {
+                ArrayList<ArrayList> outputs = []
+                buildRecord.getAttribute("outputs").split(';').collectEntries { entry ->
+                    outputs += entry.replaceAll('\\[|\\]', '').split(',')
```

Comment:
Same as in the dbb-ucd-packaging script.

Reply:
Same comment as above :)